### ISSUE TITLE
Changing the `benchmarkRepoRemoteUrl` requires a restart of the Runner

### ISSUE THREAD

**Issue by sgraf812** [IPDSnelting/velcom#295]
At least *I think* that is what I was observing. I got strange setup errors:

```
#############
## General ##
#############
Runner name:     i44pc31
Hash:            4f55f2c49316d7f318a42dea0c24fc6eb32fe13e
System:          Linux amd64 4.15.0-117-generic
CPU:             Intel(R) Core(TM) i5-8500 CPU @ 3.00GHz (6 threads)
Memory:          15785 MiB total, 14906 MiB available
Java version:    11.0.11 by Ubuntu
User:            velcom
Bench repo hash: 658df3afff5833772e2d52deea2b9cc592635be5

#################
## Setup error ##
#################
`bench` script not found or not readable
```

I monitored the what the runner did and realised that it never saved a `bench_repo`. Quite strange.

I haven't taken the time yet to determine whether the issue really is reproducible.

**Comment by Garmelon** [comment]
If I remember correctly, the runner stores the current bench repo hash but not its remote url. However, changing the repo should also change the hash (unless it has the same contents), so I don't know what's happening here.

I assume you checked whether the bench repo has an executable `bench` script at hash 658df3afff5833772e2d52deea2b9cc592635be5.

**Comment by Garmelon** [comment]
Just to clarify some more: After changing the backend config file, the backend *always* needs to be restarted. The "Refetch all repos" button on the queue page is not enough in this case.

**Comment by sgraf812** [comment]
> I assume you checked whether the bench repo has an executable bench script at hash 658df3afff5833772e2d52deea2b9cc592635be5.

Yes, I checked it. Indeed it was the same as e.g. the has from http://speedcenter.informatik.kit.edu/velcom/run-detail/87215517-c461-4baf-92c1-30d0010cb890, which checks out (I tried using the same bench repo for testing reasons).

Meanwhile I've switched and updated the bench repo multiple times without restarting the runner, so it appears it was a fluke. Closing as invalid.

Thanks for holding my hand, though!
